api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -41,4 +42,12 @@ public static String cdrVersionToId(DbCdrVersion cdrVersion) {
   public static Short dataAccessLevelToStorageEnum(DataAccessLevel dataAccessLevel) {
     return CommonStorageEnums.dataAccessLevelToStorage(dataAccessLevel);
   }
+
+  public static String cdrVersionToEtag(int cdrVersion) {
```

Comment:
[please change] I wouldn't put int-to-string mappings in CommonMappers, because you might get false positives with other types. I.e.  MapStruct doesn't know that these are etag-specific.

Reply:
I've added qualifiers to the names:
`@Named("cdrVersionToEtag")` and `@Named("etagToCdrVersion")`
to navigate this problem. @ericsong  is this reasonable or bad practice?

Reply:
That looks like a great solution to me.